community/front-end/ofe/infrastructure_files/gcs_bucket/webserver/startup.sh

```diff
@@ -159,6 +159,10 @@ EOF
 # Install go version specified in go.mod file
 #
 GO_VERSION=$(awk '/^go/ {print $2}' "/opt/gcluster/hpc-toolkit/go.mod")
+# Check if the version string has only two numbers separated by dots
```

Comment:
Is there any case where it might only have the major version (single integer) instead of both major and minor versions?

Reply:
Even if technically yes (which may not even be the case), I think no. The version is coming from our go.mod file. We would never not specify the minor version.